src/test/java/com/wikia/webdriver/testcases/renametool/RenameToolTests.java

```diff
@@ -204,4 +202,53 @@ public void NewUserCreateEditMessageWallAndRenameDone() {
     Assertion.assertEquals(driver.getCurrentUrl(), expectedWallUrl);
     wall.verifyMessageEditTextRenameDone(title, message, newName);
   }
+
+  @Test
+  public void phalanxBlocksForbiddenPhrase() {
+    String username = "QArename";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "QArename" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)